cppwinrt/component_writers.h

```diff
@@ -988,9 +988,22 @@ namespace winrt::@::implementation
     static void write_generated_static_assert(writer& w)
     {
         auto format = R"(
-// Note: Remove this static_assert after copying these generated source files to your project.
-// This assertion exists to avoid compiling these generated source files directly.
-static_assert(false, "Do not compile generated C++/WinRT source files directly");
+// WARNING: This file is automatically generated from IDL. Do not directly
```

Comment:
Incorrect. This file is generated from the winmd. Instead of saying how the file is generated, just say that it was generated.

```suggestion
// WARNING: This file is automatically generated by a tool. Do not directly
```